test/CMakeLists.txt

```diff
@@ -32,31 +32,33 @@ target_include_directories(TestForall
 blt_add_test( NAME TestForall
               COMMAND TestForall )
 
-blt_add_executable( NAME TestLoopFuser
-                    SOURCES LoopFuserTest.cxx
-                    DEPENDS_ON ${care_test_dependencies} )
+if (CARE_ENABLE_LOOP_FUSER)
```

Comment:
At the very bottom of this file there is a test called Benchmarks. That also uses the loop fuser and needs to be protected. (Eventually we'll want to make it a proper benchmark, but that is outside the scope of this pull request.)

Reply:
Changed, thanks good catch